tests/Feature/Submission/Tests/TestXMLTest.php

```diff
@@ -89,5 +90,57 @@ public function testValidXML(): void
                 ],
             ],
         ]);
+
+        DatabaseCleanupUtils::removeBuild($this->project->builds()->firstOrFail()->id);
+
+        $this->submitFiles($this->project->name, [
```

Comment:
Can you move this to a separate test case in this file so it's easier to debug if things fail?